src/api/dutchx.ts

```diff
@@ -176,6 +175,15 @@ async function init(): Promise<DutchExchange> {
     return dx.depositAndSell(t1, t2, amount, { from: account })
   }
 
+  depositAndSell.call = async (
```

Comment:
unnecessary `async`. Also I think it became removable from many functions here after the recent refactor.